en_ewt-ud-dev.conllu

```diff
@@ -14703,13 +14703,13 @@
 # newpar id = newsgroup-groups.google.com_alt.animals.cat_0382388c4d68ddf5_ENG_20031019_125000-p0002
 # text = - U P D A T E D -
 1	-	-	PUNCT	NFP	_	2	punct	2:punct	_
-2	U	update	VERB	VBN	Tense=Past|VerbForm=Part	0	root	0:root	_
-3	P	p	X	GW	_	2	goeswith	2:goeswith	_
-4	D	d	X	GW	_	2	goeswith	2:goeswith	_
-5	A	a	X	GW	_	2	goeswith	2:goeswith	_
-6	T	t	X	GW	_	2	goeswith	2:goeswith	_
-7	E	e	X	GW	_	2	goeswith	2:goeswith	_
-8	D	d	X	GW	_	2	goeswith	2:goeswith	_
+2	U	updated	VERB	VBN	Tense=Past|Typo=Yes|VerbForm=Part	0	root	0:root	_
```

Comment:
Technically I don't think this is a typo, but rather a stylistic choice to break the word. That said it's probably simpler to use `Typo=Yes` for all `goeswith` as currently required in the spec. @dan-zeman agreed?

Reply:
@nschneid I think it was you who proposed that `goeswith` always implies `Typo=Yes` :-) I just implemented it in the validator once it was approved by the core group. And I cannot imagine how the validator could discriminate, so yes, agreed.